### ISSUE TITLE
_changes call never happens

### ISSUE THREAD

**Issue by veetil09** [cloudant-labs/cloudant-follow#27]
@smithsz 

- *cloudant-follow version:* `0.16.1`
- *Node version:* `v6.9.4`
- *A small code sample that demonstrates the issue.*:

```Javascript
const follow = require('cloudant-follow');
const opts = {};
const feed = new follow.Feed(opts);
feed.db            = "https://<myaccount>.cloudant.com/<db_name>/_changes";
feed.since         = "now"; // just for testing
feed.heartbeat     = 3600  * 1000;
feed.inactivity_ms = 86400 * 1000;
feed.headers = {
'Authorization': <bearer xyz>
};

// Event Fired
feed.on('confirm_request', function(req) {
   console.log('requets sent to DB');
});

// Event Fired
feed.on('start', function() {
    console.log('starting');
});

// Not fired since no response since `db_response` never seem to be called
feed.on('confirm', function(dbobj) {
    console.log('confirmed');
});

// Never fired even though I simulated a document update in Cloudant DB
feed.on('change', function(change) {
    console.log('Doc ' + change.id + ' in change ' + change.seq + ' is neither stinky nor ugly.');
});

// Event fired
feed.on('error', function(er) {
    console.error('Since Follow always retries on errors, this must be serious');
    throw er;
});

feed.follow();

```
Have already tested the cloudant `_changes` API for the same DB with Postman and it works fine.
Or am I doing something wrong here?

**Comment by smithsz** [comment]
Hi @veetil09,
Your `feed.db` parameter needs to be the fully qualified database URL, namely `https://<myaccount>.cloudant.com/<db_name>`. Don't include the `/_changes`.
Hopefully this solves the problem. Let me know if you're still seeing errors.

**Comment by ricellis** [comment]
TLDR; there is no advantage

`cloudant-follow` exists primarily for the purposes of providing fixes on top of the `follow` project that it forks. This library is consumed by `nodejs-cloudant` and Apache nano to provide their `db.follow` functionality.

**Comment by veetil09** [comment]
@ricellis @smithsz Thanks for the information. I will use the `nodejs-cloudant` version then.
